src/main/java/racingCar/RacingCarUtils.java

```diff
@@ -8,4 +8,10 @@
     public static int randomValueGenerator() {
         return new Random().nextInt(BOUNDARY);
     }
+
+    public static void checkCarNameRule(String car) {
```

Comment:
```suggestion
    public static void checkCarNameRule(String carName) {
```

carName과 같은 명확한 이름을 파라미터 이름으로 사용하면 어떨까요?